python/openinference-instrumentation/README.md

```diff
@@ -8,3 +9,72 @@ Utility functions for OpenInference instrumentation.
 ```shell
 pip install openinference-instrumentation
 ```
+
+## Customizing Spans
+
+This core `openinference-instrumentation` package offers convenient tools to read & write from the OTEL Context. The OTEL context is then used to propagate attributes to every span created. We do this using Python context managers:
+
+* `using_session`: to specifiy a session ID.
+* `using_user`: to specifiy a user ID.
+* `using_metadata`: to add custom metadata.
+* `using_tag`: to add tags.
+* `using_prompt_template`: to reflect the prompt template used.
+* `using_attributes`: to specify all the above at once.
```

Comment:
give the rationale as to why you would want to set these. Readers might not know why or how they might want to use these

Reply:
Done